### PR TITLE
transfer_file_over_ipv6: To catch correct error info

### PR DESCRIPTION
**Pull request by yanan-fu**
The old design match wrong md5 value if execute md5sum command failed

id: 1713595
Signed-off-by: Yanan Fu <yfu@redhat.com>

### CODE REVIEW

**Comment by yanan-fu** [comment]
@pezhang  call for your review, thanks !

**Comment by yanan-fu** [comment]
@xiywang

**Comment by pezhang** [comment]
Test pass.
 (1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.bigfile_transfer_over_ipv6.q35: PASS (636.16 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.bigfile_transfer_over_ipv6.q35: PASS (473.60 s)

Looks good to me.
Acked-by: Pei Zhang pezhang@redhat.com

File: qemu/tests/transfer_file_over_ipv6.py [hunk 1]
```diff
@@ -40,11 +40,10 @@ def get_file_md5sum(file_name, session, timeout):
...
-        try:
-            o = session.cmd_output("md5sum %s" % file_name, timeout=timeout)
-            file_md5sum = re.findall(r"\w+", o)[0]
-        except IndexError:
-            test.error("Could not get file md5sum in guest")
+        s, o = session.cmd_status_output("md5sum %s" % file_name, timeout=timeout)
+        if s != 0:
+            test.error("Get file md5sum failed as %s" % o)
+        file_md5sum = re.findall(r"\w{32}", o)[0]
         return file_md5sum
```
[the remark this answers is not in the document]
Comment (yanan-fu) [reply qemu/tests/transfer_file_over_ipv6.py]: Good,  I follow the old design, didn't though make `file_md5sum`  as the return value directly, will update it.
And for `if s`, if it is not must for you,  i prefer `if s != 0` to check the return code,  please let me keep  it, thanks for your review.

**Comment by yanan-fu** [comment]
@ybduan updated, please help review again, thanks a lot!

**Comment by ybduan** [comment]
LGTM
Acked-by: Yanbin Duan yduan@redhat.com

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Pull request merged**